### ISSUE TITLE
AI objects

### ISSUE THREAD

**Issue by cklasacrcez** [Picoseconds/sanctuary#18]
I do not see the logic responsible for intelligent AI objects such as Wolfs or Cows which interact with player or I may be wrong so correct me. I would try to implement it.

**Comment by Picoseconds** [comment]
Yes, we do not currently have animals. You are welcome to create a PR to try and implement them. A good start would be in [bundle.js](https://moomoo.io/bundle.js), which has complete (pseudo-)server code for animals.

**Comment by Picoseconds** [comment]
@Flappy6969 have you made any progress on your implementation?

**Comment by Flappy6969** [comment]
busy playing LoL XD

**Comment by cklasacrcez** [comment]
For now I have written an algorithm for turrets, water, respawn packet for bulls, wolfs etc, and I have improved the arrows

**Comment by Picoseconds** [comment]
Are you making correct commit messages with commitzen for every change?

**Comment by itzTheMeow** [comment]
any more progress on this?

**Comment by Picoseconds** [comment]
I have an experimental patch, I'll add it to a new branch

**Comment by itzTheMeow** [comment]
gg thanks man

**Comment by itzTheMeow** [comment]
when animals

**Comment by itzTheMeow** [comment]
> For now I have written an algorithm for turrets, water, respawn packet for bulls, wolfs etc, and I have improved the arrows
@cklasacrcez wheres this at

**Comment by itzTheMeow** [comment]
when animals

**Comment by Picoseconds** [comment]
No.
